cypress/src/components/Header.js

```diff
@@ -2,21 +2,27 @@ import ContactUs from '../pages/ContactUs';
 import SignIn from '../pages/SignIn';
 
 export default class Header {
+  headerContainerLocator = '#header';
 
-    headerContainerLocator = '#header';
-
-    get contactUsButton() {
-        return cy.get(this.headerContainerLocator).contains('a', 'Contact us');
-    }
-    get signInButton(){
-        return cy.get(this.headerContainerLocator).contains('a','Sign in');
-    }
-    clickOnContactUsButton() {
-        this.contactUsButton.click();
-        return new ContactUs();
-    }
-    clickOnSignInButton(){
-        this.signInButton.click();
-        return new SignIn();
-    }
+  get contactUsButton() {
+    return cy.get(this.headerContainerLocator).contains('a', 'Contact us');
+  }
+  get signInButton() {
+    return cy.get(this.headerContainerLocator).contains('a', 'Sign in');
+  }
+  get nameAccount() {
+    return cy.get(this.headerContainerLocator).contains('John Wick');
```

Comment:
`.contains('John Wick'); `
mut be in method 
`checkUserNameIsPresent()`
Also, don't use the hardcode approach. You should refactor like this:
`checkUserNameIsPresent(userName)`